app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -19,17 +19,18 @@ def sign_class(widget)
     widget.percentual_change > 0 ? 'u-plus' : 'u-minus'
   end
 
-  def dashboard_collection_link(singular_name, collection, path, plural = nil, icon_name = nil)
+  def dashboard_collection_link(singular_name, collection, path, icon_name: nil, plural: nil, notice: false)
     link_to path, class: 'DashboardNavigation-link' do
       link_text = pluralize(number_to_human(collection.size), singular_name, plural)
       link_text = link_text.prepend "#{icon(icon_name)} " if icon_name
+      link_text = link_text.prepend ".u-notice" if notice
```

Comment:
I do not think this is OK you want to prepend the `notice` variable not `'.u-notice'` which looks like a class attribute

Reply:
Ah notice is a boolean.
So you probably want to add it to `:class` https://github.com/3scale/porta/pull/279/files#diff-2ef0db8cb2db1089bb506d22b9615e69R23

Reply:
OMG you are right, I think what @thomasmaas asked me was to add the class to the element so I just updated the method.